index.html

```diff
@@ -249,6 +249,45 @@ <h2 class="work-title-props">Multi-Post<br/>Stories</h2>
 
             </section>
 
+            <!-- About Myself -->
+            <section class="flex-box vertical flex-center transparent-bg black-fg" id="section_1" >
+
+                <h1>About Myself</h1>
+
+                <p>Hello I'm a software developer! I can help you build a product, feature or website.<br/>Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesite to contact me.</p>
+
```

Comment:
- [ ] The line break (`br`) here is not necessary as the original design does not have one. It would be better to remove it and simply align the text to the left.